README.md

```diff
@@ -73,40 +73,90 @@ A filtering function for modifying a tile's features and properties to support l
+    - For now, only the first item in the array will be processed; the rest are discarded. (*TODO in the future*: expand support for more than one worldviews.)
+  -`params.worldview_property` **String** the name of the property that specifies in which worldview a feature belongs.
+    - Default value: `worldview`.
+    - The vector tile encoded property must contain a single ISO 3166-1 alpha-2 country code or a comma-separated string of country codes that define which worldviews the feature represents (for example: `JP`, `IN,RU,US`).
+  - `params.worldview_prefix` **String** prefix for the worldview property.
+    - Default value: `_mbx_`.
+  - `params.worldview_default` **String** default worldview to assume when `params.worldviews` is not provided.
+    - Default value: `US`.
+  - `params.class_property` **String** the name of the property that specifies the class category of a feature.
+    - Default value: `class`.
+  - `params.class_prefix` **String** prefix for the class property.
+    - Default value: `_mbx_`.
 - `callback` **Function** callback function that returns `err` and `buffer` parameters
 
+The existence of the parameters `params.languages` and `params.worldviews` determines the type of features that will be returned:
+
+- Non-localized feature: when `params.languages` and `params.worldviews` both do not exist.
+  - No new language property.
+  - The property `{language_property}` retains its original value.
+  - Properties like `{language_property}_{language}` are kept.
+  - Properties like `{language_prefix}{language_property}_{language}` are dropped.
+  - All features with `{worldview_property}` are kept.
+  - All features with `{worldview_prefix}{worldview_property}` are dropped except for those that have the value `all`.
+  - Property `{class_property}` retains its original value.
+  - Property `{class_prefix}{class_property}` is dropped.
+
+- Localized feature: when either `params.languages` or `params.worldviews` exists.
+  - A new `{language_property}_local` property is created to keep the original value of `{language_property}`
+  - The value of `{language_property}` is replaced with the first translation found by looping through `params.languages`.
+    - First searches for `{language_property}_{language}` and then `{language_prefix}{language_property}_{langauge}` before moving on to the next language in `params.languages`.
+  - Properties like `{language_property}_{language}` are dropped.
+  - Properties like `{language_prefix}{language_property}_{language}` are dropped.
+  - All features with `{worldview_property}` are dropped except for those that have the value `all`.
+  - Features with `{worldview_prefix}{worldview_property}` are kept if their `{worldview_prefix}{worldview_property}` value is
+    - `all`, or
+    - a comma-separated list that contains the first item of `parmas.worldviews`, in which case the value of `{worldview_prefix}{worldview_property}` is replaced by that one single worldview country code.
```

Comment:
note to self: so the final tile will have a property like `_mbx_worldview`?

Reply:
Nope the final tile does not have `_mbx_worldview`. Corrected this sentence in 3711e05918f95ff3494302a0c36e07fe1bdfad08.